### DISCUSSION TITLE
【教程】如何更换自己的头像

### DISCUSSION THREAD

**Discussion by 566001** [share121/inter-knot#1936] [其他]
首先鼠标左键单击箭头所指的位置【头像】
![屏幕截图 2024-08-13 205210](https://github.com/user-attachments/assets/3739b7ef-36be-4b94-8ee8-130afe719a3a)
然后再次点击箭头所指的位置
![屏幕截图 2024-08-13 205227](https://github.com/user-attachments/assets/92f0c7ac-4eeb-4cf4-b323-4b2ef267a563)
点击蓝色圆圈里的编辑
![屏幕截图 2024-08-13 205251](https://github.com/user-attachments/assets/ce8355f0-efbe-46e0-9eab-205cb78bb985)
点击红色框里的【Upload a photo…】
![屏幕截图 2024-08-13 205325](https://github.com/user-attachments/assets/e2e8c1c9-994a-48b7-b7b0-df6b509e0fc7)
然后选择照片并打开
【注意图片不能大于1M】
稍微等一下加载，会让你选取照片范围，裁剪后点击【设置新的头像】
最后往下滑点击绿色的按钮【更新个人资料】
![屏幕截图 2024-08-13 210705](https://github.com/user-attachments/assets/2af1784a-409a-463b-a8eb-1a876620a427)
之后刷新一下网页就成功了

**Comment by L502567** [comment]
但凡你发早一点，我也不必如此大费周章

**Comment by Fuklos** [comment]
大佬，出现这串英文怎么搞“Something went really wrong and we can’t process that picture. ”

**Reply by L502567** [reply]
意思是你的照片有问题，无法处理

**Reply by 566001** [reply]
是不是图片太大了？也有可能是图片有缺失，或者出现违规内容

**Reply by L502567** [reply]
有可能，这种情况我没遇到过，不太清楚
